src/components/feed/feed.jsx

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+/* 
+ @author [Jenny Doctor](https://github.com/jendoc)
+*/
+
+export const Feed = ({ children }) => {
+  return <div className="bg-alpha" role={"feed"}>{children}</div>;
+};
+
+const Heading = styled.h3`
+  font-weight: 400;
+  font-size: 10px;
```

Comment:
where possible, it's better to use the bootstrap className font-size-x (replace x with whichever size you want) so that way we don't have random usages of pixel-based font sizes in a lot of places. Same with font-weight

Reply:
Where are the bootstrap sizes defined in our project? I tried to use `.fs-x` but the lowest I can go is 6, and it's still a bit big.